src/app/main/component/ubs/components/ubs-submit-order/ubs-submit-order.component.ts

```diff
@@ -176,14 +175,16 @@ export class UBSSubmitOrderComponent extends FormBaseComponent implements OnInit
         )
         .subscribe(
           (response) => {
+            const { orderId, link } = JSON.parse(response);
```

Comment:
because I use " orderId " from response to pass it  into:

--182--  this.ubsOrderFormService.transferOrderId(orderId) ; 
and 
--187-- this.localStorageService.setUbsFondyOrderId(orderId) ;